### ISSUE TITLE
lxc container not working

### ISSUE THREAD

**Issue by Eddiecz** [openwrt/luci#1762]
flashed 
turris-omnia-sysupgrade.img.gz | 21f37514940697f9d88f9069748c555e0d217c2464cfc0998f7d79f95e57cb26 | 5101.1 KB | Mon Apr 23 17:26:02 2018

Firmware Version | OpenWrt SNAPSHOT r6745-b1f5dd3 / 			LuCI Master (git-18.112.36203-82abf9a)

Installed luci and luci-app-lxc

Containers URL is images.linuxcontainers.org , but no containers shown. Changed to uk.images.linuxcontainers.org , same result.

**Comment by localnet** [comment]
Are you waiting enough time to download the image list? Can you try to execute the next command from the console?

```
sh /usr/share/lxc/templates/lxc-download --list --no-validate --server images.linuxcontainers.org
```

**Comment by Eddiecz** [comment]
I tried command `sh /usr/share/lxc/templates/lxc-download --list --no-validate --server images.linuxcontainers.org`  that was okay.
The last state was that the containers appeared, but is impossible create container in luci.
I've tried lxc-create command, it ended in error, so it's probably a problem in lxc itself.
Now i can not test, turris is  back on original firmware.
Openwrt has DSA, original swconfig, but I need Vlan and tagging.
